build.gradle

```diff
@@ -46,6 +46,14 @@ dependencies {
     testImplementation group: 'junit', name: 'junit', version: '4.12'
 }
 
+tasks.register("buildCloudFunctionArchive", Zip) {
+    archiveClassifier = shadowJar.archiveClassifier
+    destinationDirectory = jar.destinationDirectory
+    from(jar.destinationDirectory) {
+        include "*.jar"
+    }
+}
```

Comment:
how much does zipping the jar really gain us, since the jar is already compressed? Don't we have to unzip it again before deploying?